pkg/gotohelm/transpiler.go

```diff
@@ -1051,6 +1051,8 @@ func (t *Transpiler) transpileCallExpr(n *ast.CallExpr) Node {
 		return &Selector{Expr: args[0], Field: "AsMap"}
 	case "github.com/redpanda-data/helm-charts/pkg/gotohelm/helmette.UnmarshalInto":
 		return args[0]
+	case "github.com/redpanda-data/helm-charts/pkg/gotohelm/helmette.MustDecodeYamlInto":
+		return args[0]
```

Comment:
I can not find `MustDecodeYamlInto` function in `helmette` package.

Reply:
I'll double-check - this got revitalised in a rebase I think.